### ISSUE TITLE
Server Crash/ Infinate Restarts Because of Draconic Evolution

### ISSUE THREAD

**Issue by tyler489** [SpongePowered/SpongeForge#1617]
-SpongeForge version: 1.11.2-2315-6.1.0-BETA-2410
- Forge version: 2315
- Java version: 8
- Operating System:

 - [x] Linux (Ubuntu, Debian, Fedora, etc)
 - [ ] macOS
 - [ ] Windows

- Plugins/Mods: 

**What is the expected result?**
Server Stops and restart loop
**What is the current result?**
Server doesnt stop and restart

https://pastebin.com/CSAk8uWj
Disabling Draconic Evolution and BRandons core Allows server to start

**Comment by gabizou** [comment]
01.07 04:03:32 [Multicraft] Skipped 275 lines due to rate limit (30/s)


Get the actual server log, not the console log from multi craft...

**Comment by tyler489** [comment]
Here is a plugins list 
01.07 11:02:45 [Server] Server thread/INFO Plugins (182): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, SpongeAPI, SpongeForge, Aroma1997Core, Micdoodle8 Core, OpenEye, Backpacks!, FoamFixCore, ccl-entityhook, Actually Additions, Baubles, extrautils2, AngelRing 2 Bauble, Animal Bikes, AppleCore, Just Enough Items, AppleSkin, Armor Overlay, Shadowfacts' Forgelin, Mantle, Chisel, Tinkers' Construct, TESLA, TheDragonLib, ArmorPlus, Aroma1997Core|Helper, Aroma1997sDimension, Astral Sorcery, Psi, Quark, AutoRegLib, BD Lib, Bed Bugs, Bedrock-B-Gone, Better Achievements, Better Builder's Wands, BiblioCraft, Biomes O' Plenty, BiblioWoodBoP, Natura, BiblioWoodNatura, Cyclic, Guide-API, Blood Magic: Alchemical Wizardry, Bookshelf, Bookshelf, Botania, SonarCore, Calculator, MrCrayfish's Furniture Mod, Chameleon, CodeChicken Lib, ChickenChunks, Chisels & Bits, IndustrialCraft 2, Compact Solar Arrays, CompactStorage, Computality, Cooking for Blockheads, CraftTweaker JEI Support, Crafttweaker, CXLibrary, Cyberware, Cyclops Core, Dark Utilities, ElecCore, DeepResonance, Storage Drawers, Drawers & Bits, Elevator Mod, Embers, EnderStorage, Ender Utilities, Ender Zoo, Valkyrie Lib, Environmental Tech, EvilCraft, Zero CORE, Extreme Reactors, EZStorage 2, Fire's Random Things, FL, FluxNetworks, FoamFix, Not Enough Items, FTBLib, Galacticraft Core, Galacticraft Planets, Advanced Generators, GlareTorch, Gravestone Mod, Grue, Hammer Core, Hardcore Darkness, HelpFixer, The One Probe, Immersive Engineering, Immersive Petroleum, Tesla Core Lib, Industrial Foregoing, Inventory Pets, Inventory Tweaks, Iron Chest, JourneyMap, LimeLib, Mekanism, MekanismGenerators, MekanismTools, Metamorph, Mike Dongles, Minecoprocessors, examplemod, Missing Pieces, MmmMmmMmmMmm, mob_grinding_utils, MTLib, Mod Tweaker, Mysterious Miscellany, Mystical Agriculture, NoMoreRecipeConflict, NotEnoughIDs, Not Enough Wands, NuclearCraft, OMLib, Open Modular Turrets, Ore Control, Ore Dictionary Converter, OreExcavation, Ore Shrubs, Overloaded, Pam's HarvestCraft, PlusTiC, Quick Leaf Decay, Random Utilities, Random Things, Ranged Pumps, Reborn Core, reborncore-MCMultiPart, Redstone Paste, Refined Storage, Reliquary, RFTools, Roguelike Dungeons, Roots 2, Rough Mobs, Silent Lib, Silent's Gems, Simple Teleporters, Simply Conveyors & More, SlashBlade, Slurpie's Dongles, Solar Flux Reborn, Soul Shards - The Old Ways, The Spice of Life, Statues mod, Steve's Carts 2, Super Crafting Frame, Tails, Tinkers Tool Leveling, Tiny Progressions, TOP Addons, TorchMaster, Translocator, WanionLib, Woot, XNet, BetterChunkLoader, Buycraft, Can I Join Now?, CatClearLag, Discord Bridge, GriefPrevention, LuckPerms, Morpheus, Nucleus, NuVotifier, WorldEdit, denseores, UniDict

**Comment by johnfg2610** [comment]
it would appear forge is overwritting something that drac evo is attempting to cast to. ive attached the full server log
[2017-07-01-7.txt](https://github.com/SpongePowered/SpongeForge/files/1116972/2017-07-01-7.txt)

**Comment by tyler489** [comment]
According to john. Brandon core has no issues by itself but with draconic evolution installed that happens.

**Comment by gabizou** [comment]
Riddle me surprised, yet another fake world implementation that triggers Sponge's tracking system. There's only an `isRemote` check that differentiates between a server sided world (which should always be a `WorldServer`), and a "remote" world, in some cases `WorldClient` or other fake worlds that shouldn't be considered for ticking or processing as if it is a server world. This isn't going to be fixed immediately as fake worlds tend to require large amounts of testing to get working correctly in Sponge (due to the limited amount of resources available to determine if a world is a fake world or not). That being said, I'm open for discussion on the matter with the developer, but I cannot "fix" the issue in SpongeForge alone.

Long story short: Not a bug in Sponge, looks like a fake world is triggering Sponge's tracking stuff by using the wrong boolean values to mark the world as not a server world. Nothing much sponge can do at this time.

**Comment by tyler489** [comment]
How can draconic evolution fix this on there end to make it compatible with sponge if they wish to

**Comment by tyler489** [comment]
Fixed in newest versions of draconic evo
